### ISSUE TITLE
Fill by gradients not supported

### ISSUE THREAD

**Issue by jershell** [DevSrSouza/svg-to-compose#5]
Hi, In the beginning, thx for the cool converter. 
I discovered filling color is null if used any gradient. For example, you can take this SVG and convert it to kt

```svg
<svg height="150" width="400">
    <defs>
        <linearGradient id="grad1" x1="0%" y1="0%" x2="100%" y2="0%">
            <stop offset="0%" style="stop-color:rgb(255,255,0);stop-opacity:1" />
            <stop offset="100%" style="stop-color:rgb(255,0,0);stop-opacity:1" />
        </linearGradient>
    </defs>
    <ellipse cx="200" cy="70" rx="85" ry="55" fill="url(#grad1)" />
</svg>
```

```kt
public val ImagesGroup.Gradient: ImageVector
    get() {
        if (_gradient != null) {
            return _gradient!!
        }
        _gradient = Builder(
            name = "Gradient", defaultWidth = 400.0.dp, defaultHeight = 150.0.dp,
            viewportWidth = 400.0f, viewportHeight = 150.0f
        ).apply {
            path(
                fill = null,
                /* expected
                          Brush.linearGradient(
                                              colors = listOf(Color.Yellow, Color.Red),
                                              start = Offset(0f, 0f),
                                              end = Offset(Float.POSITIVE_INFINITY, 0f)
                                          ),
                */
                stroke = null,
                strokeLineWidth = 0.0f,
                strokeLineCap = Butt,
                strokeLineJoin = Miter,
                strokeLineMiter = 4.0f,
                pathFillType = NonZero
            ) {
                moveTo(115.0f, 70.0f)
                arcToRelative(85.0f, 55.0f, 0.0f, true, false, 170.0f, 0.0f)
                arcToRelative(85.0f, 55.0f, 0.0f, true, false, -170.0f, 0.0f)
                close()
            }

        }
            .build()
        return _gradient!!
    }

private var _gradient: ImageVector? = null
```

**Comment by MarioFerreiraStorytel** [comment]
@jershell  I've created a pull request for this: https://github.com/DevSrSouza/svg-to-compose/pull/6

**Comment by jershell** [comment]
I tested PR. Look like a good job! =)
Screenshot from my app 
![image](https://user-images.githubusercontent.com/1238301/126710655-bd24b03e-9acd-43d0-875e-65622df3a991.png)

**Comment by MarioFerreiraStorytel** [comment]
Glad it worked, I know the formating is pretty ugly, but I'm not sure how to handle that

**Comment by DevSrSouza** [comment]
The PR was merged

**Comment by jershell** [comment]
When will the next release be?

**Comment by DevSrSouza** [comment]
Published as 0.8.0
